packages/host/android/consumerSettings.gradle

```
@@ -0,0 +1,22 @@
+import org.gradle.initialization.DefaultSettings
+
+
+ext.applyNodeAPISettings = { DefaultSettings settings ->
+    def searchDirectory = rootDir.toPath()
+    do {
+        def p = searchDirectory.resolve("node_modules/react-native")
+        if (p.toFile().exists()) {
+            println "[RN-NAPI] Found React Native in ${p.toRealPath().toString()}"
```

As per https://nodejs.medium.com/renaming-n-api-to-node-api-27aa8ca30ed8 we want to respect the Node.js project decision and use "Node-API" instead of "N-API" or "NAPI".

```suggestion
            println "[Node-API] Found React Native in ${p.toRealPath().toString()}"
```